wrapanapi/systems/rhevm.py

```diff
@@ -252,16 +252,20 @@ def _get_state(self):
     @property
     def ip(self):
         """
-        Returns IPv4 or global IPv6 address of the VM/instance
+        Returns IPv4 or global IPv6 addresses of the VM
         """
-        link_local_prefix = 'fe80::'
+        potentials = []
         for ip in self.all_ips:
-            if link_local_prefix not in ip[:len(link_local_prefix)]:
-                return ip
-        return None
+            if not ip.startswith('fe80::'):
+                potentials.append(ip)
+        return potentials or None
```

Comment:
this property returns one element everywhere. I'd rather not to deviate from expected value here.
Special cases aren't special enough to break the rules. (c)

Reply:
:man_facepalming: